### PR TITLE
REVIEW: def_minerals()

### PR DESCRIPTION
**Pull request by bethany-j-allen**
Hi @LewisAJones, I reordered the return columns and added a link to Mindat, where the URLs lead for a more extensive definition. But otherwise just a few typos, looks good!

### CODE REVIEW

File: R/def_minerals.R [hunk 1]
```diff
@@ -9,35 +9,39 @@
...
 #'   "feldspar") to return a definition for.
 #' @param element \code{character}. The chemical element abbreviation of a
-#'   mineral to return a definition for (e.g., "CaCO3"). Note: abbreviations
+#'   mineral (e.g., "CaCO3") to return a definition for. Note: abbreviations
 #'   are case sensitive.
 #'
 #' @return A \code{data.frame} containing the following columns:
 #' \itemize{
-#'   \item \code{mineral_id}: The unique identification number of the mineral.
+#'   \item \code{mineral_id}: The unique identification number(s) of the
```
Comment (LewisAJones) [inline R/def_minerals.R:17]: Does this need the (s) here and below (L17 & L18)? I guess you will only ever have one number in each row?

File: R/def_minerals.R [hunk 2]
```diff
@@ -9,35 +9,39 @@
...
-#'   \item \code{hardness_min}: The minimum hardness value (Moh's hardness
-#'   scale) of the mineral.
-#'   \item \code{hardness_max}: The maximum hardness value (Moh's hardness
-#'   scale) of the mineral.
-#'   \item \code{mineral_color}: The color description of the mineral.
-#'   \item \code{lustre}: The lustre description of the mineral.
-#'   \item \code{crystal_form}: The crystal form of the mineral.
 #'   \item \code{formula}: The chemical formula of the mineral.
 #'   \item \code{formula_tags}: The chemical formula of the mineral with
 #'   appropriate sub/superscript html tags.
```
Comment (LewisAJones) [inline R/def_minerals.R:23]: Will made the point that we should indent when text flows overs more than one line to ease reading, e.g.:

```suggestion
#'     appropriate sub/superscript html tags.
```

**Review by LewisAJones — APPROVED** [review]
Thanks for the review Bethany! Including the link to mindat is a good shout. I just have two small comments (see review).

File: R/def_minerals.R (hunk 1 above)
Comment (bethany-j-allen) [reply R/def_minerals.R:17]: True! Agreed, these can be left out.

File: R/def_minerals.R (hunk 2 above)
Comment (bethany-j-allen) [reply R/def_minerals.R:23]: Sorry, I keep forgetting :)

**Review by LewisAJones — APPROVED** [review]
Hi @bethany-j-allen, looks good to me! I just added an extra space to some of the documentation so it indents properly.

**Pull request merged**